std_msgs/Header header
float64 e_reader            # e_reader reading
float64 e_reader_variance   # e_reader_variance reading
